### DISCUSSION TITLE
Welcome to ShedShield General Discussions!

### DISCUSSION THREAD

**Discussion by ZJouba** [ZJouba/ShedShield#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Harshcombo50548** [comment]
Hello, ZJouba I wanted to contact you but did not know how so this is the only way I can think of, I was wondering if you could help me I Have a website that is not very complicated  ( it is run on cloudflare pages with my own git repo) I want to know how can I get information about (todays) loadshedding times and then display it on my website without using a server so I can just use my scripts.js folder it should update every hour to check that the loadshedding times have not changed any help will be appreciated

**Comment by ZJouba** [comment]
Hey @Harshcombo50548 here's the API page
https://eskomsepush.gumroad.com/l/api

**Reply by Harshcombo50548** [reply]
I atemted to integrate the EskomSePush API for load shedding times, I encountered a Cross-Origin Resource Sharing (CORS) restriction. While I understand the potential solution involving a Proxy Server, I prefer to explore alternatives without running an additional server My website is hosted on my domain through Cloudflare Pages, utilizing my own Git repository. The primary goal is to provide my friends with real-time updates on the server's status and inform them about load shedding schedules  the websites whole point is to check the status (online or offline) of my Minecraft server and dynamically adjusts the background and displayed mobs
